### PR TITLE
Suppress freddy output in RabbitMQ adapter

### PR DESCRIPTION
**Pull request by SteamRiC**

### CODE REVIEW

File: lib/logasm/adapters/rabbitmq_adapter.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+require_relative '../loggers/null_logger'
```
Comment (indrekj) [inline lib/logasm/adapters/rabbitmq_adapter.rb]: require this directly in the logasm.rb. It is simple enough to be required everywhere

Comment (indrekj) [inline lib/logasm/adapters/rabbitmq_adapter.rb]: Usually if you start requiring with `../` prefix then it means that something is not correct

File: lib/logasm/loggers/null_logger.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+class Logasm
+  module Loggers
```
Comment (indrekj) [inline lib/logasm/loggers/null_logger.rb]: can we move this to adapters instead?

**Comment by urmastalimaa** [comment]
+1

File: logasm.gemspec [hunk 3]
```diff
@@ -4,7 +4,7 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
 Gem::Specification.new do |gem|
   gem.name          = "logasm"
-  gem.version       = '0.2.2'
+  gem.version       = '0.2.3'
```
Comment (indrekj) [inline logasm.gemspec]: please create a separate commit for version bumps. commit message "Bump version to 0.2.3". This way it is easier to follow the commit history

Comment (indrekj) [inline logasm.gemspec]: and you forgot to "bundle"

**Pull request merged**